docs/sources/reference/components/prometheus.exporter.catchpoint.md

```diff
@@ -0,0 +1,110 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/prometheus.exporter.catchpoint/
+description: Learn about prometheus.exporter.catchpoint
+title: prometheus.exporter.catchpoint
+---
+
+# prometheus.exporter.catchpoint
+
+The `prometheus.exporter.catchpoint` component embeds
+[catchpoint_exporter](https://github.com/grafana/catchpoint-prometheus-exporter) for collecting statistics from a Catchpoint account.
+
+## Usage
+
+```alloy
+prometheus.exporter.catchpoint "<LABEL>" {
+    port              = PORT
+    verbosity_logging = <VERBOSITY_LOGGING>
+    webhook_path      = <WEBHOOK_PATH>
+}
+```
+
+## Arguments
+
+The following arguments can be used to configure the exporter's behavior.
+Omitted fields take their default values.
+
+| Name                | Type     | Description                                           | Default                 | Required |
+| ------------------- | -------- | ----------------------------------------------------- | ----------------------- | -------- |
+| `port`              | `string` | The account to collect metrics for.                   | `"9090"`                | no       |
```

Comment:
"The account to collect metrics for." sounds off here, is there a one to one relation between port and account?

Reply:
This was incorrect, I fixed the descriptions!